cli/cmd/pro.go

```diff
@@ -51,14 +52,27 @@ var proCmd = &cobra.Command{
 }
 
 func executeRemoteUpdateToken(ctx context.Context, client *kube.Client, namespace string, onPremToken string) error {
-	uiSvcProxyEndpoint := fmt.Sprintf("/api/v1/namespaces/%s/services/%s:%d/proxy/api/token/update/%s", namespace, k8sconsts.OdigosUiServiceName, k8sconsts.OdigosUiServicePort, onPremToken)
-	request := client.Clientset.RESTClient().Get().AbsPath(uiSvcProxyEndpoint).Do(ctx)
-	_, err := request.Raw()
-	if err != nil {
-		return err
-	} else {
-		return nil
+	uiSvcProxyEndpoint := fmt.Sprintf(
+		"/api/v1/namespaces/%s/services/%s:%d/proxy/api/token/update",
+		namespace,
+		k8sconsts.OdigosUiServiceName,
+		k8sconsts.OdigosUiServicePort,
+	)
+
+	tokenPayload := fmt.Sprintf(`{"onprem-token": "%s"}`, onPremToken)
+	body := bytes.NewBuffer([]byte(tokenPayload))
```

Comment:
Consider using go built in json package. You can also share the struct between the CLI and the frontend via the common module